modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -374,6 +379,16 @@ class Session implements ISession {
         // -- file porter config
         this.filePorter = new FilePorter(this)
 
+        // -- fsSize used when provisioning OFS resources
+        if (config.fsSizeTb) {
+            try {
+                this.fsSize = Integer.parseInt(config.fsSizeTb.toString())
```

Comment:
the fuck is this language? why u go to string? what was it before?

Reply:
its a generic `Object` type